apps/api/src/application/controller/WebSocketController.ts

```diff
@@ -0,0 +1,26 @@
+import type { WSContext, WSEvents } from 'hono/ws';
+
+export const ws_stocks = new Map();
+
+export function WebSocketHandler(stock_id: string): WSEvents {
```

Comment:
```suggestion
export const WebSocketHandler = (stock_id: string): WSEvents => {
```
他のところでこっちの記法を使用しているのでこっちで書いて欲しいです！